core/services/relay/evm/mercury_provider.go

```diff
@@ -96,6 +98,10 @@ func (p *mercuryProvider) ReportCodecV3() relaymercuryv3.ReportCodec {
 	return p.reportCodecV3
 }
 
-func (p *mercuryProvider) ContractTransmitter() relaymercury.Transmitter {
+func (p *mercuryProvider) ContractTransmitter() ocrtypes.ContractTransmitter {
+	return p.transmitter
+}
+
+func (p *mercuryProvider) MercuryServerFetcher() relaymercury.MercuryServerFetcher {
```

Comment:
for my own curiosity why did the name change? (i don't understand much about mercury nor ocr2)